carbonmark/components/Layout/AddressSection/index.tsx

```diff
@@ -1,8 +1,10 @@
-import { CopyAddressButton } from "@klimadao/lib/components";
+import { urls } from "@klimadao/lib/constants";
 import { Domain } from "@klimadao/lib/types/domains";
-import { concatAddress } from "@klimadao/lib/utils";
+import { concatAddress, isTorusProvider, useWeb3 } from "@klimadao/lib/utils";
 import { Trans } from "@lingui/macro";
+import { CopyAddressButton } from "components/CopyAddressButton";
```

Comment:
my best guess is that changing this import instead of `@klimadao/lib/components` is what broke the `CopyAddressButton` styles

Reply:
You're right, I should have caught this. Fixed in [e5c55ae](https://github.com/KlimaDAO/klimadao/pull/1321/commits/e5c55ae2278f0e525a0d0b5d59dd812de31c98bd)

Reply:
You're right, I should have caught this. Fixed in [e5c55ae](https://github.com/KlimaDAO/klimadao/pull/1321/commits/e5c55ae2278f0e525a0d0b5d59dd812de31c98bd)